avocado/utils/crypto.py

```diff
@@ -17,6 +17,8 @@
 import logging
 import os
 
+LOG = logging.getLogger('avocado.test')
```

Comment:
Is there a preferred way to access this logger? For instance, how does this type of accessing it compare to
```
from avocado.core.output import LOG_JOB
```
that some utilities have done before?

Reply:
Probably we should use the global one `LOG_JOB` when we are sure that  'avocado.test' is our namespace. However I decided to keep as it is because I think we should change this in the future for `__name__` as described here #4997.